src/components/screens/sprint-view-screen.tsx

```diff
@@ -1,11 +1,270 @@
+                {strings.sprint.allTasks}
+              </MenuItem>
+            </Select>
+          </FormControl>
+          <Card sx={{ margin: 0, width: "100%", height: "100", marginBottom: "16px" , marginTop: "16px", padding: "0px",
+            '& .negative-value': {
+              color: 'red',
+            }}}>
+            <DataGrid
+              sx={{
+                borderTop: 0,
+                borderLeft: 0,
+                borderRight: 0,
+                borderBottom: 0,
+                '& .header-color': {
+                  backgroundColor: '#f2f2f2',
+                }
+              }} 
+              autoHeight={true}
+              localeText={{ noResultsOverlayLabel: strings.sprint.notFound }}
+              disableColumnFilter
+              hideFooter={true}            
+              rows={allocations}
+              columns={[
+                { 
+                  field: 'projectName',  
+                  headerClassName: 'header-color',                
+                  filterable: false,
+                  headerName: strings.sprint.myAllocation, 
+                  flex: 2, valueGetter: (params) => getProjectName(params.row),
+                  renderCell:(params) => <><Box minWidth="45px" style={{marginRight:"10px"}} component="span" sx={{ bgcolor: getProjectColor(params.row), height: 25, borderRadius: "5px"}} />{getProjectName(params.row)}</>
+                },
+                { 
+                  field: 'allocation', 
+                  headerClassName: 'header-color',
+                  headerName: strings.sprint.allocation, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(totalAllocations(params.row))
+                },
+                { 
+                  field: 'timeEntries', 
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.timeEntries, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(getTotalTimeEntries(params.row) || 0),
+                },
+                { 
+                  field: 'allocationsLeft', 
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.allocationLeft, 
+                  flex: 1, cellClassName: (params) =>  timeLeft(params.row) < 0 ? "negative-value" : "", valueGetter: (params) => getHoursAndMinutes(timeLeft(params.row))
+                },
+              ]}             
+            />
+            <Box sx={{ backgroundColor:"#e6e6e6", display: 'flex', justifyContent: "space-between", padding: "5px", paddingTop:" 10px", paddingBottom:" 10px"}}>
+              <span style={{paddingLeft: "5px", color: unallocatedTime(allocations) < 0 ? "red" : ""}}>{strings.sprint.unAllocated} {getHoursAndMinutes(unallocatedTime(allocations))} </span> 
+              <span style={{ paddingRight:"5px"}}>
+                {strings.sprint.currentSprint}: {sprintStartDate.toLocaleString()} - {sprintEndDate.toLocaleString()}
+              </span>
```

Comment:
This could all be done with a formatted string and passing the variables to the Strings library rather than having to combine localized strings and colons and dashes as is being done now.
Here is another reference of something similar being done:
https://github.com/Metatavu/oss-surveys-management/blob/53a92e960114a0a5752f282fedc155db252785f7/src/components/screens/overview-screen.tsx#L161